core/src/main/java/org/ehcache/core/spi/cache/Store.java

```diff
@@ -446,6 +450,19 @@
      */
     void initStore(Store<?, ?> resource);
 
+    /**
+     * Gets the internal ranking for the {@link Store} instances provided by this {@code Provider} of the store's
+     * ability to handle the specified resources.  A higher rank value indicates a more capable {@code Store}.
+     *
+     * @param resourceTypes the set of {@code ResourceType}s for the store to handle
+     * @param serviceConfigs the collection of {@code ServiceConfiguration} instances that may contribute
+     *                       to the ranking
+     *
+     * @return a non-negative rank indicating the ability of a {@code Store} created by this {@code Provider}
```

Comment:
No real reason ... I just wanted a value of 0 to mean the provider didn't provide support.